docs/dev/new_check_howto.md

```diff
@@ -318,6 +366,20 @@ Our check sends a Service Check though, so we need to add it to the `service_che
 ]
 ```
 
+Find below the description for each attributes -each one of them is mandatory- of your `service_checks.json` file:
```

Comment:
no spaces around em-dashes, i.e.:
Find below the description for each attributes—each one of them is mandatory—of your `service_checks.json` file: